docs/resources/user_attributes.md

```diff
@@ -0,0 +1,27 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "satori_user_attributes Resource - terraform-provider-satori"
+subcategory: ""
+description: |-
+  User Satori Attributes.
+---
+
+# satori_user_attributes (Resource)
```

Comment:
What do you think if we will call the resource satori_user? 
In future we can add more staff there....

Reply:
satori_user_setting